wbooks/src/app/components/SingleBook/styles.ts

```diff
@@ -0,0 +1,44 @@
+import { StyleSheet } from 'react-native';
+import { colors } from '@constants/colors';
+
+export const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.white,
+    borderRadius: 5,
+    elevation: 1,
+    height: 90,
+    justifyContent: 'center',
+    marginHorizontal: '5.33%',
+    marginTop: 118,
+    shadowColor: colors.black,
+    shadowOffset: { height: 1, width: 0 },
+    shadowOpacity: 1,
+    shadowRadius: 1
+  },
+  contentWrapper: {
+    alignItems: 'flex-start',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    paddingLeft: '5.33%'
```

Comment:
Check if you need all of these styles:
* `flex-start` is the default
* `paddingLeft: '5.33%'` => NEVER
* Merge this style with `container`

Reply:
Refactored